src/main/java/com/quantumlytangled/gravekeeper/GraveKeeperConfig.java

```diff
@@ -34,44 +34,39 @@
   public static int EXPIRE_TIME_SECONDS = 7200;
   public static boolean INSTANT_FOREIGN_COLLECTION = false;
   public static boolean OWNER_ONLY_COLLECTION = false;
-
-  private static String[] CHARM_ARMOR_HELD_NAMES = new String[]{
-      "twilightforest:charm_of_keeping_1"
-  };
-  private static String[] CHARM_ARMOR_HOTBAR_NAMES = new String[]{
-      "twilightforest:charm_of_keeping_2"
-  };
-  private static String[] CHARM_FULL_NAMES = new String[]{
-      "twilightforest:charm_of_keeping_3"
-  };
-
   public static List<Item> CHARM_ARMOR_HELD_ITEMS = null;
   public static List<Item> CHARM_ARMOR_HOTBAR_ITEMS = null;
   public static List<Item> CHARM_FULL_ITEMS = null;
   public static List<Item> CHARM_ITEMS = null;
-
   public static boolean ANY_ENCHANT_IS_SOULBOUND = false;
   public static boolean MOVE_SOULBOUND_ITEMS_TO_MAIN_INVENTORY = true;
-  private static String[] SOULBOUND_ENCHANTMENT_NAMES = new String[]{
-      "enderio:soulbound",
-      "cofhcore:soulbound",
-      "aoa3:intervention"
-  };
-  public static String[] SOULBOUND_TAG_VALUE = new String[]{
-      "Botania_keepIvy:1",
-      "spectreAnchor:0"
+  public static String[] SOULBOUND_TAG_STRINGS = new String[]{
+      "{ \"Botania_keepIvy\": \"1\" }",
+      "{ \"spectreAnchor\": \"0\" }"
   };
   public static int KEEP_SOULBOUND_AMOUNT = 5;
-
   public static List<Enchantment> SOULBOUND_ENCHANTMENTS = null;
-  public static Map<String, Boolean> SOULBOUND_TAGS = null;
-
+  public static List<NBTTagCompound> SOULBOUND_TAGS = null;
   public static int SEARCH_MIN_ALTITUDE = 0;
   public static int SEARCH_RADIUS_ABOVE_M = 10;
   public static int SEARCH_RADIUS_BELOW_M = -1;
   public static int SEARCH_RADIUS_HORIZONTAL_M = 5;
   public static int SPAWN_DIMENSION_ID = 0;
   public static int USE_BED_OR_SPAWN_LOCATION_BELOW_Y = 0;
+  private static String[] CHARM_ARMOR_HELD_NAMES = new String[]{
```

Comment:
any reasons to move Charms configuration away from their values?

Reply:
Sorry for that, formatted the file and it seems I checked "reorder entries".